conf.py

```diff
@@ -351,8 +351,11 @@ def xapian_run_example_command(ex):
         return "%s -unsafe -target:exe -out:%s.exe %s -r:XapianSharp.dll\n./%s.exe" \
             % (csc, ex, xapian_code_example_filename(ex), ex)
     elif highlight_language == 'java':
-        javac = get_tool_name('JAVAC', 'javac')
-        java = get_tool_name('JAVA', 'java')
+        java_bindings_dir = get_tool_name('JAVA_BINDINGS_DIR', 'usr/local/') # change default
```

Comment:
I'd be inclined to use `os.environ.get`, and if it's `None` then you don't need to set classpath and so on explicitly. (In case someone has done something clever with packaging it into Java system directories).